managedtenants/bundles/addon_bundles.py

```diff
@@ -140,7 +140,7 @@ def _validate_config(self, config):
         except jsonschema.exceptions.ValidationError as e:
             raise AddonBundlesError(f"schema validation error for {self}: {e}")
 
-    @lru_cache()
+    @lru_cache(maxsize=128)
```

Comment:
nit: can this fix be separated to it's own commit in this PR? Just a little cleaner git history IMO.

Reply:
sure thing, fixed